core/worker/pom.xml

```diff
@@ -71,6 +71,18 @@
       <groupId>org.osgi</groupId>
       <artifactId>org.osgi.core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.service.component.annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.annotation</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.service.metatype.annotations</artifactId>
+    </dependency>
```

Comment:
You can also remove:
```xml
    <dependency>
      <groupId>org.apache.sling</groupId>
      <artifactId>org.apache.sling.commons.osgi</artifactId>
    </dependency>
```

Reply:
No, I can't - WebDriverHelper uses PropertiesUtil from this package.

Reply:
Maybe -> `return StringUtils.defaultString(properties.get(name), defaultValue);` at https://github.com/Cognifide/aet/blob/a024722d58c9704720199560a19ae4897058f555/core/worker/src/main/java/com/cognifide/aet/worker/drivers/WebDriverHelper.java#L39

Reply:
Is this method used (`WebDriverHelper.getProp`)? Some of this method references were removed from `ChromeWebDriverFactory.activate`.